### PR TITLE
feat: DIA-1064: Get all / sample tasks for prompts task list

### PR DESCRIPTION
**Pull request by hakan458**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [x] Backend (API)
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: label_studio/ml_models/models.py [hunk 1]
```diff
@@ -110,6 +110,7 @@ class ModelRun(models.Model):
     class ProjectSubset(models.TextChoices):
         ALL = 'All', _('All')
         HASGT = 'HasGT', _('HasGT')
+        SAMPLE = 'Sample', _('Sample')
```
Comment (matt-bernstein) [inline label_studio/ml_models/models.py:113]: Given that we plan to remove this field eventually and instead parameterize other subset types with something like `sample_n_tasks=`, what's the best migration plan for that in the future? Mark `Sample` as deprecated somehow or alias it to `All(sample_n_tasks=20)` without deleting it?

Comment (pakelley) [reply label_studio/ml_models/models.py:113]: Yeah, I'd say just deprecating it in the future is fine, personally

**Review by pakelley — APPROVED** [review]

File: label_studio/ml_models/models.py (hunk 1 above)
Comment (hakan458) [reply label_studio/ml_models/models.py:113]: I think we will still use this subset, and provide an additional subset size param, which I just added to the LSE PR

Comment (matt-bernstein) [reply label_studio/ml_models/models.py:113]: this is implicitly a subset of `All`, but what if we wanted a subset of hasGT, or a subset of hasSubmitted or hasAccepted?

**Review by wesleylima — APPROVED** [review]

**Pull request merged**